.env

```
@@ -0,0 +1,8 @@
+ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiNjdjMDYzOWQ4MzZlZThiZTcwNjE5MGE5Iiwicm9sZSI6ImFkbWluIiwiaWF0IjoxNzYwMTk5MzQ5fQ.E61wk2cgsIEC0IihKE8vVS999r0OWh88WIE3e94F_6o"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)